.github/workflows/build.yml

```diff
@@ -4,6 +4,22 @@ on:
   pull_request
 
 jobs:
+  run-workflow:
+    name: PR Workflow
```

Comment:
nit: a more descriptive name would be helpful

Reply:
I used `PR Workflow` to make it consistent with the other repos https://github.com/aws/aws-sam-cli/blob/develop/.github/workflows/build.yml#L11 and https://github.com/aws/aws-lambda-builders/blob/develop/.github/workflows/build.yml#L10. But I can change that to something like `Parent PR Status Check`.

Reply:
Also out of curiosity, why is this necessary? I would've thought the workflow fails anyway if any of the steps fail?

Reply:
This is for easy maintenance of branch protection rules. Currently each check has to be manually added as a rule. But with a parent rule, new rules can just be added to the GH action and they don't need to be added in the branch protection rules manually.